bot/src/index.ts

```diff
@@ -12,20 +13,33 @@ const DEFAULT_REFETCH_INTERVAL = 60 * 1000;
 const SETUP_DELAY = 3 * 1000;
 const REFETCH_INTERVAL = parseInt(process.env.REFETCH_INTERVAL ?? '') || DEFAULT_REFETCH_INTERVAL;
 
-const loop = async function (network: string): Promise<void> {
+const loopCollateral = async function (network: string): Promise<void> {
     try {
         const activeAuctions = await getAllAuctions(network);
         if (activeAuctions.length === 0) {
             return;
         }
         const newAuctions = getNewAuctionsFromActiveAuctions(activeAuctions);
-        newAuctions.map(notify);
+        newAuctions.map(notifyCollateral);
         participate(network, activeAuctions);
     } catch (error) {
         console.error('loop error:', error);
     }
 };
 
+const loopSurplus = async function (network: string): Promise<void> {
```

Comment:
Can you please move those functions into the appropriate files (`surplus.ts` vs `collateral.ts`)?